README.md

```diff
@@ -11,71 +12,111 @@ SEOTools is a package for **Laravel 5+** and **Lumen** that provides helpers for
 
 [![Latest Stable Version](https://poser.pugx.org/artesaos/seotools/v/stable)](https://packagist.org/packages/artesaos/seotools) [![Total Downloads](https://poser.pugx.org/artesaos/seotools/downloads)](https://packagist.org/packages/artesaos/seotools) [![Latest Unstable Version](https://poser.pugx.org/artesaos/seotools/v/unstable)](https://packagist.org/packages/artesaos/seotools) [![License](https://poser.pugx.org/artesaos/seotools/license)](https://packagist.org/packages/artesaos/seotools)
 
-## Features
-- Friendly Interface
-- Ease of set titles and meta tags
-- Ease of set metas for twitter and opengraph
+Features
+--------
```

Comment:
Why change H2 heading definition style? (here and further)

Reply:
This does not change document HTML representation, but significantly improves plain-text representation making source '.md' more readable.